libretroshare/src/util/rsnet.cc

```diff
@@ -40,6 +43,72 @@
 	#define BIG_ENDIAN  4321
 #endif
 
+//https://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml#dns-parameters-2
+#define DNSC_IN    1 //Internet (IN)
```

Comment:
Please use `constexpr` instead of `#define`, we don't need the preprocessor to fiddle with that things just the compiler to know the exact value

Reply:
Done.